src/components/LoansYouMightLike/MinimalLoanCard.vue

```diff
@@ -16,7 +16,7 @@
 					:loan-id="loan.id"
 					:name="loan.name"
 					class="small-text name"
-				>{{ loan.name }}</p>
+				>{{ loan.name | truncate(20) }}</p>
```

Comment:
Also, please add a title attribute with the full name to this element.

Reply:
@mcstover PR updated with requested changes.